tests/queries/0_stateless/02715_exponential_smoothing_alpha.sql

```diff
@@ -0,0 +1,42 @@
+/* exponentialSmoothingAlpha tests */
+SELECT exponentialSmoothingAlpha()(value) over (ORDER BY timestamp ASC) from (SELECT number as timestamp, number * 2 - (number % 7 - 3) / 7 as value from numbers_mt(100)) OFFSET 99; -- { serverError BAD_ARGUMENTS }
+SELECT exponentialSmoothingAlpha(0.5, 0.5)(value) over (ORDER BY timestamp ASC) from (SELECT number as timestamp, number * 2 - (number % 7 - 3) / 7 as value from numbers_mt(100)) OFFSET 99; -- { serverError BAD_ARGUMENTS }
+SELECT exponentialSmoothingAlpha(-1)(value) over (ORDER BY timestamp ASC) from (SELECT number as timestamp, number * 2 - (number % 7 - 3) / 7 as value from numbers_mt(100)) OFFSET 99; -- { serverError BAD_ARGUMENTS }
+SELECT exponentialSmoothingAlpha(2)(value) over (ORDER BY timestamp ASC) from (SELECT number as timestamp, number * 2 - (number % 7 - 3) / 7  as value from numbers_mt(100)) OFFSET 99; -- { serverError BAD_ARGUMENTS }
+SELECT exponentialSmoothingAlpha(0.5)(value, timestamp) over (ORDER BY timestamp ASC) from (SELECT number as timestamp, number * 2 - (number % 7 - 3) / 7  as value from numbers_mt(100)) OFFSET 99; -- { serverError BAD_ARGUMENTS }
+SELECT exponentialSmoothingAlpha(0.5) over (ORDER BY timestamp ASC) from (SELECT number as timestamp, number * 2 - (number % 7 - 3) / 7  as value from numbers_mt(100)) OFFSET 99; -- { serverError BAD_ARGUMENTS }
+SELECT exponentialSmoothingAlpha(0)(value) over (ORDER BY timestamp ASC) from (SELECT number as timestamp, number * 2 - (number % 7 - 3) / 7  as value from numbers_mt(100)) OFFSET 99;
+SELECT exponentialSmoothingAlpha(0.2)(value) over (ORDER BY timestamp ASC) from (SELECT number as timestamp, number * 2 - (number % 7 - 3) / 7  as value from numbers_mt(100)) OFFSET 99;
+SELECT exponentialSmoothingAlpha(0.5)(value) over (ORDER BY timestamp ASC) from (SELECT number as timestamp, number * 3 - (number % 7 - 3) / 7  as value from numbers_mt(100)) OFFSET 99;
```

Comment:
Why are we using different values here (`number * 2` in l. 9 vs. `number * 3` in l. 10)? I think the results will be easier to verify when all queries use the same data.

EDIT: Thinking about it, maybe some manual test data will be sufficient already and more intuitive. E.g. CREATE a table, insert (e.g.) 6 rows manually, then run the queries with different alpha against that data. We would not need an OFFSET clause because there is only few rows. But because of that, it will also be easy to verify that everything is correct by looking at the expected results file.